packages/terra-date-picker/tests/nightwatch/components/DatePickerExcludeDates.jsx

```diff
@@ -1,12 +1,17 @@
 import React from 'react';
+import Base from 'terra-base';
 import DatePicker from '../../../lib/DatePicker';
 
+const locale = document.getElementsByTagName('html')[0].getAttribute('lang');
+
 const DatePickerExcludeDates = () => (
-  <DatePicker
-    name="date-input"
-    excludeDates={['2017-04-01', '2017-04-03']}
-    selectedDate={'2017-04-02'}
-  />
+  <Base locale={locale}>
+    <DatePicker
+      name="date-input"
+      excludeDates={['2017-04-01', '2017-04-03']}
```

Comment:
On PR #671, we had this discussion regarding the consistency of the date format: 

![screen shot 2017-08-01 at 10 24 57 am](https://user-images.githubusercontent.com/14099737/28833029-c9539748-76a3-11e7-819b-efead7d868e0.png)

I think it would be important to include/consider how this affects `excludeDates`, as well as `selectedDate`.